src/App.tsx

```diff
@@ -102,6 +105,17 @@ const App: React.FC = () => {
                       <Route path={Routes.TREE} exact component={TreePage} />
                       <AuthRedirect from={Routes.HOME} />
                       <AuthRedirect from={Routes.SETTINGS} />
+                      <Route
+                        path={Routes.SITE}
+                        exact
+                        render={({ match }) => {
+                          return (
+                            <AuthRedirect
+                              from={`${ParameterizedRouteBases.SITE}${match.params.id}`}
+                            />
+                          );
+                        }}
+                      />
```

Comment:
This is to pass the path params to the redirect as well, otherwise the redirect would go back to `/:id`, which would cause an error